app/core/serializers.py

```diff
@@ -55,10 +55,16 @@ def create(self, validated_data):
                     'user_exists_error': True
                 }
                 raise res
-            user = get_user_model().objects.create_user(email=user_info.get('email'), password=user_info.get('password'))
-            UserSerializer(data=user_info).update(user, user_info)
-        recruiter = Recruiter.objects.create(user=user, company=validated_data['company'], is_admin=validated_data['is_admin'])
-        recruiter.save()
+            user = get_user_model().objects.create_user(**user_info)
+            recruiter = Recruiter.objects.create(user=user, **validated_data)
+        else:
```

Comment:
We could move this else block to `if not user_info:` above and avoid the nested if blocks on line 50.

Reply:
Updated 👍